hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/group/GroupService.java

```diff
@@ -39,13 +43,15 @@ public Group getGroupDetails(String groupName) {
         return groupRepository.getGroupDetails(groupName);
     }
 
-    public String createGroup(Group group) {
+    public String createGroup(Group group, Principal createdBy) {
```

Comment:
just thinking, should we contain occurrence of `Principal` within the endpoints? i'm in two minds, what are your reasons to do it?